### PR TITLE
EAS 424 -  Make up "Sign up"

### PR DESCRIPTION
**Pull request by IrinaEndeladze**

### CODE REVIEW

File: components/contents/SignUp.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+                className: classes.inputStyle,
+              }}
+            />
+          </div>
+
+          <TextField
+            id="filled-basic"
+            label="E-mail"
+            variant="filled"
+            type="text"
```
Comment (NikitoriusMinsk) [inline components/contents/SignUp.js]: type "email"

File: components/contents/SignUp.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+            InputProps={{
+              className: classes.inputStyle,
+            }}
+          />
+          <TextField
+            id="filled-basic"
+            label="Phone number"
+            variant="filled"
+            type="number"
```
Comment (NikitoriusMinsk) [inline components/contents/SignUp.js:145]: dont use type "number" for inputting a phone

File: components/contents/SignUp.js [hunk 3]
```diff
@@ -103,20 +153,52 @@ export const SignUp = () => {
...
+                    onMouseDown={handleMouseDownPassword}
+                    edge="end"
+                  >
+                    {values.showPassword ? <VisibilityOff /> : <Visibility />}
+                  </IconButton>
+                </InputAdornment>
+              ),
             }}
           />
           <TextField
```
Comment (NikitoriusMinsk) [inline components/contents/SignUp.js:176]: Separate inputs into components

**Review by NikitoriusMinsk — CHANGES_REQUESTED** [review]
The pages should be full width.
Move css files to styles folder

**Review by NikitoriusMinsk — CHANGES_REQUESTED** [review]
please fix previous comments

File: components/contents/SignUp.js (hunk 3 above)
Comment (NikitoriusMinsk) [reply components/contents/SignUp.js:176]: i mean move the textfield with its props into a separate component to make it easier to read and reuse

File: pages/confirmPasswordPage/index.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import Head from "next/head";
+import { PasswordConfirm } from "../../components/contents/PasswordConfirm";
+
+function ConfirmPasswordPage() {
+  return (
+    <div>
+      <Head>
+        <title>Confrim password</title>
+      </Head>
+      <PasswordConfirm />
```
Comment (NikitoriusMinsk) [inline pages/confirmPasswordPage/index.js:10]: A page is a component by itself and shouldn't just return a single component

File: pages/confirmPasswordPage/index.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import Head from "next/head";
+import { PasswordConfirm } from "../../components/contents/PasswordConfirm";
+
+function ConfirmPasswordPage() {
+  return (
+    <div>
```
Comment (NikitoriusMinsk) [inline pages/confirmPasswordPage/index.js:6]: use fragments (<> </>) to wrap instead of <_div>

File: pages/createNewPass/index.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
+import Head from "next/head";
+import { CreateNewPass } from "../../components/contents/CreateNewPass";
+
+function PasswordRecoveryPage() {
+  return (
+    <div>
+      <Head>
+        <title>Create new password</title>
+      </Head>
+      <CreateNewPass />
```
Comment (NikitoriusMinsk) [inline pages/createNewPass/index.js:10]: A page is a component by itself and shouldn't just return a single component

**Review by NikitoriusMinsk — CHANGES_REQUESTED** [review]

File: pages/confirmPasswordPage/index.js (hunk 4 above)
Comment (IrinaEndeladze) [reply pages/confirmPasswordPage/index.js:10]: I did it because of Links.  I don't understand how to modify this?

Comment (NikitoriusMinsk) [reply pages/confirmPasswordPage/index.js:10]: The <PasswordConfirm/> shouldn't be a separate component, please just put its code into the page file

**Pull request merged**